### DISCUSSION TITLE
This is really nice

### DISCUSSION THREAD

**Discussion by Deed8855** [Yewchi/vulft#26] [General]
Good job these bots are best ive ever seen.

**Comment by Yewchi** [comment]
Thanks for the feedback, I haven't looked at this project in quite a while and I'd honestly be surprised if the bots still worked without some patched in bug.

I did my best to build them resilient and dynamic, but things can change, sometimes API functions are removed, things beyond my control.

I wasn't getting much feedback on steam and even found I was not able to update the project at all on the workshop with the dev tools. So the project has been put on hiatus.

However, again, I appreciate the feedback. Thank you. :)
